bugsnag-plugin-android-ndk/src/main/jni/utils/serializer.c

```diff
@@ -340,8 +339,6 @@ void bsg_serialize_app(const bsg_app_info app, JSON_Object *event_obj) {
 
 void bsg_serialize_app_metadata(const bsg_app_info app, JSON_Object *event_obj) {
   json_object_dotset_string(event_obj, "metaData.app.activeScreen", app.active_screen);
```

Comment:
From your comments on the PR:

> Note: app.activeScreen and device.osBuild have been left in place as they affect the context/runtimeversions respectively so should be kept in the struct.

Is this consistent if the value is written to `metaData.app. activeScreen`? Can we not move it to metadata everywhere?

Reply:
Also, should we lower-case `metaData`?